plugins/ipfs/local/localipfs.go

```diff
@@ -42,9 +44,16 @@ var GetAttrList testbedi.GetAttrListFunc
 func init() {
 	NewNode = func(dir string, attrs map[string]string) (testbedi.Core, error) {
 		mdns := false
+		binary := ""
 
-		if _, err := exec.LookPath("ipfs"); err != nil {
-			return nil, err
+		var ok bool
+
+		if binary, ok = attrs["binary"]; !ok {
+			var err error
```

Comment:
do we need this?

Reply:
Ya, `binary` is defined outside of the block, so we can't define the error inline.